ast/dml.go

```diff
@@ -1686,21 +1686,32 @@ func (n *UpdateStmt) Accept(v Visitor) (Node, bool) {
 type Limit struct {
 	node
 
-	Count  ExprNode
-	Offset ExprNode
+	Count                ExprNode
+	Offset               ExprNode
+	ExplicitOffsetSyntax bool
```

Comment:
We could just always restore as `LIMIT x OFFSET y`, and remove this field.

Reply:
I think it's better to keep the restored SQL as what it is originally, so we may keep this field for both `LIMIT row_count  OFFSET offset ` syntax and `LIMIT offset, row_count`  syntax

Reply:
`Restore()` by design is allowed to change the output as long as they are equivalent. So I'd prefer not to complicate the structure.

Reply:
OK. I wonder if there's situation when we need the original SQL exactly.